Apromore-Core-Components/Apromore-Portal/src/main/webapp/openModelInBPMNio.zul

```diff
@@ -405,6 +405,8 @@
             zk.Widget.$('$slider').setCurpos(newFontSize);
             Apromore.BPMNEditor.app.editor.changeFontSize(newFontSize);
           }
+
+          Apromore.BPMNEditor.qbpProcessMaxLimit = (${arg.qbpProcessMaxLimit})?${arg.qbpProcessMaxLimit}:25000;
```

Comment:
Can you also add the change from https://github.com/apromore/ApromoreEE/pull/1702? This fixes an issue where `qbpProcessMaxLimit` is not found when we open models in publish mode.